monkestation/code/modules/clothing/gloves/power_gloves.dm

```diff
@@ -0,0 +1,96 @@
+/obj/item/clothing/gloves/color/yellow/power_gloves
+	name = "Power Gloves"
+	desc = "Insulated gloves with onboard machinery that appears to be able to redirect the electrical current towards a creature."
+	armor_type = /datum/armor/power_gloves
+	var/datum/action/cooldown/spell/pointed/glove_zap/zap = new
+
+/obj/item/clothing/gloves/color/yellow/power_gloves/Destroy()
+    qdel(zap)
+    return ..()
+
+/datum/armor/power_gloves
+	acid = 50
+	bio = 50
+	fire = 80
+
+/datum/action/cooldown/spell/pointed/glove_zap
+	name = "Nerd Blast"
+	desc = "Blast your foes with the electricity surging beneath your feet!"
+	button_icon_state = "lightning"
+	cooldown_time = 10 SECONDS
+	spell_max_level = 1
+	sparks_amt = 4
+	spell_requirements = SPELL_REQUIRES_HUMAN
+	antimagic_flags = NONE
+	background_icon_state = ACTION_BUTTON_DEFAULT_BACKGROUND
+	overlay_icon_state = "bg_default_border"
+
+//had to recreate tesla zap into a pointed version
+/datum/action/cooldown/spell/pointed/glove_zap/proc/target_tesla_zap(atom/source, atom/target, power, zap_flags = ZAP_DEFAULT_FLAGS, max_damage = 90)
+	// damage the shock causes, DO NOT fuck with the function unless you know what you're doing.
+	var/shock_damage = (zap_flags & ZAP_MOB_DAMAGE) ? (min(round( 5*(ROOT (4, (power)))), max_damage) + rand(-5, 5)) : 0
+	var/mob/living/electrocute_victim = target
+
+	source.Beam(target, icon_state="lightning[rand(1,12)]", time = 5) //Creates lightning beam
+	var/zapdir = get_dir(source, target)
+	if(zapdir)
+		. = zapdir
+
+	if(power >=10000000000) //Dust if over 10 gw power :]
+		electrocute_victim.dust(TRUE, FALSE, TRUE)
+
+	if (isliving(target))
+		electrocute_victim.electrocute_act(shock_damage, source, 1, SHOCK_TESLA | ((zap_flags & ZAP_MOB_STUN) ? NONE : SHOCK_NOSTUN))
+
+	if(issilicon(target)) //sillycons get emp'd
+		var/mob/living/silicon/S = target
```

Comment:
one letter var

Reply:
Pretty sure this is a standard var for a specific silicon. Energy ball uses it and I've seen it pretty often when referencing a specific silicon target